kafka-starter-testing/src/main/java/org/galatea/kafka/starter/testing/avro/AvroPostProcessor.java

```diff
@@ -95,8 +103,8 @@ public void registerType(LogicalType type, AvroFieldType fieldType) {
       }
       message.put(field.pos(), fieldType.defaultValue());
     } else if (primitiveType.equals(Type.RECORD)) {
```

Comment:
Won't this create a NPE? You can't do `<null>.equals()`, right?? Pretty sure you can use `==` for enums.

Reply:
I don't see how primitiveType could ever be null though. I think you're right we could use `==`, but I've gotten in the habit of using .equals() whenever possible